service/pixelated/adapter/services/mail_sender.py

```diff
@@ -25,13 +26,26 @@ class MailSender():
     def __init__(self, account_email_address, smtp_client=None):
         self.account_email_address = account_email_address
 
+    def recepients_normalizer(self, mail_list):
+        return set(mail_list)
+
+    def get_email_addresses(self, mail_list):
+        clean_mail_list = []
+        for mail_address in mail_list:
+            if "<" in mail_address:
+                match = re.search(r'<(.*)', mail_address)
+                clean_mail_list.append(match.group(1).strip('>'))
+            else:
+                clean_mail_list.append(mail_address)
+        return self.recepients_normalizer(clean_mail_list)
+
     def sendmail(self, mail):
         recipients = flatten([mail.to, mail.cc, mail.bcc])
-
+        normalized_recepients = get_email_addresses(recipients)
```

Comment:
There is a typo on the variable name, and it is used right below, this should be breaking